### DISCUSSION TITLE
How to contribute

### DISCUSSION THREAD

**Discussion by caimaoy** [fastapi/fastapi#4242] [Questions] [answered]
> First-time contributors need a maintainer to approve running workflows.

1. How to know who is the maintainer of the repo.
2. How to connect with the maintainer.

**Comment by elrik75** [comment]
It seems it's not actively maintained anymore: https://github.com/tiangolo/fastapi/discussions/3970

**Comment by Kludex** [comment, marked as answer]
Please check: https://fastapi.tiangolo.com/help-fastapi/
